browser/src/map/Clipboard.js

```diff
@@ -923,7 +923,7 @@ L.Clipboard = L.Class.extend({
 			L.Browser.clipboardApiAvailable = false;
 			window.prefs.set('clipboardApiAvailable', 'false');
 			// Prefetch selection, so next time copy will work with the keyboard.
-			app.socket.sendMessage('gettextselection mimetype=text/html,text/plain;charset=utf-8');
+			app.socket.sendMessage(that.getMetaURL() + '&MimeType=text/html,text/plain;charset=utf-8');
```

Comment:
This one is really strange: the socket messages typically have a syntax where a lowercase command starts the line, then we have various parameters here. Do you really send a raw URL here or do I misread this change?

Also, you don't really describe what is the problem you try to fix here. :-)

Reply:
:D Forgotten debugging thing.

Reply:
I see this is gone from the latest version, good.